requirements.txt

```diff
@@ -0,0 +1 @@
+pre-commit
```

Comment:
this is more like a dev dependency, so I usually end up putting those in requirements-dev.txt